js/src/Ice/EnumBase.d.ts

```diff
@@ -14,21 +14,6 @@ declare module "@zeroc/ice" {
              */
             constructor(name: string, value: number);
 
-            /**
```

Comment:
Enums should be compared by reference, not clear why we added these methods in the first place:

https://doc.zeroc.com/ice/3.7/language-mappings/javascript-mapping/client-side-slice-to-javascript-mapping/javascript-mapping-for-enumerations

I think is safe to remove them.